### PR TITLE
DAOS-6901 test: Re-enable osa offline drain test

### PR DESCRIPTION
**Pull request by rpadma2**
Test-tag-hw-medium: pr,hw,medium,ib2 osa

Summary:
- Re-enable the drain test.
- Make the offline drain to use IOR instead of using DAOS API methods.
- Move common code to osa_utils.py
- Test with single pool

Signed-off-by: rpadma2 <ravindran.padmanabhan@intel.com>

### CODE REVIEW

File: src/tests/ftest/util/osa_utils.py [hunk 1]
```diff
@@ -16,6 +17,14 @@
 from pydaos.raw import (DaosContainer, IORequest,
                         DaosObj, DaosApiError)
 
+try:
+    # python 3.x
+    import queue as queue
```
[the remark this answers is not in the document]
Comment (rpadma2) [reply src/tests/ftest/util/osa_utils.py:22]: Do we need to resolve python3 lint errors now? Atleast based on Test group discussion looks like we have enabled more lint errors... This should not be an issue. Several scripts have this issue.

File: src/tests/ftest/util/osa_utils.py [hunk 2]
```diff
@@ -163,6 +177,36 @@ def verify_single_object(self):
...
         self.container.close()
 
+    def run_ior_thread(self, action, oclass, api, test):
+        """Start the IOR thread for either writing or
+        reading data to/from a container.
+        Args:
+            action (str): Start the IOR thread with Read or
+                          Write
+            oclass (str): IOR object class
+            API (str): IOR API
```
Comment (shimizukko) [inline src/tests/ftest/util/osa_utils.py:187]:
```suggestion
            api (str): IOR API
```

**Review by shimizukko — APPROVED** [review]
LGTM

File: src/tests/ftest/util/osa_utils.py (hunk 2 above)
Comment (rpadma2) [reply src/tests/ftest/util/osa_utils.py:187]: Sure. I will fix it on the next PR... I have a lot of laundry list to change on osa_utils.py... I willl keep this in mind.

**Review by ravalsam — APPROVED** [review]
LGTM

**Review by shimizukko — DISMISSED** [review]
LGTM

**Review by shimizukko — APPROVED** [review]
LGTM

**Review by ravalsam — APPROVED** [review]

**Comment by rpadma2** [comment]
This PR is ready for landing... All tests passed CI testing... The offline drain tests are passing now.

**Review by byron-marohn — APPROVED** [review]

**Pull request merged**